src/D2L.CodeStyle.Analyzers/Immutability/ImmutableAttributeConsistencyChecker.cs

```diff
@@ -154,7 +154,7 @@ CancellationToken cancellationToken
 				foreach( var baseTypeSyntax in baseTypes ) {
 					TypeSyntax typeSyntax = baseTypeSyntax.Type;
 
-					ITypeSymbol thisTypeSymbol = model.GetTypeInfo( typeSyntax ).Type;
+					ITypeSymbol thisTypeSymbol = model.GetTypeInfo( typeSyntax, cancellationToken ).Type;

```

Comment:
Caught this one only because `CancellationToken cancellationToken` was a method parameter.